ts/torch_handler/image_classifier.py

```diff
@@ -4,10 +4,24 @@
 import torch
 import torch.nn.functional as F
 from torchvision import transforms
+import logging
+import os
 
 from .vision_handler import VisionHandler
 from ..utils.util  import map_class_to_label
 
+logger = logging.getLogger(__name__)
+
+ipex_enabled = False
+if os.environ.get("TS_IPEX_ENABLE", "false") == "true":
+    ipex_enabled = True
+    try:
+        import accimage
+        ipex_enabled = True
+    except ImportError as error:
+        logger.warning("Please instlal accimage for optimized preprocessing")
```

Comment:
```suggestion
        logger.warning("Please install accimage for optimized preprocessing")
```

Reply:
Hi @HamidShojanazeri , please see my below comment on usage of accimage to remove 2 aten::copy ops. And it's correct that accimage is only used in vision handler.